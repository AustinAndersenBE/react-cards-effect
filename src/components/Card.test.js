import React from 'react';
import { render } from '@testing-library/react'
import Card from './Card'

test('renders without crashing', () => {
  const card = { image: 'test_image.png', value: "JACK", suit: "SPADES" };
  render(<Card card={card} />)
});

test('snapshot test', () => {
  const card = { image: 'test_image.png', value: "JACK", suit: "SPADES" };
  const { asFragment } = render(<Card card={card} />);
  expect(asFragment()).toMatchSnapshot();
});
