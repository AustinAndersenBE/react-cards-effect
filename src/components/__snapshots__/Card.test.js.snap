// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="JACK of SPADES"
      src="test_image.png"
    />
  </div>
</DocumentFragment>
`;
